Breeding-Insight/bi-web
# [BI-1364] View a shared ontology that a program is subscribed to

# Description
**Story:** [BI-1364 View a shared ontology that a program is subscribed to](https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?modal=detail&selectedIssue=BI-1364)

A Vuex module was created to access subscrition state data on mutiple pages.
Batch import and New Trait buttons are conditionally rendered depending on subscription status.
The ontology tab is conditionally rendered for Import page and Germplasm tab is now active by default when navigating to Import page.

# Dependencies
bi-api BI-1364 branch

# Testing

1. Create three programs: a sharer, a subscriber, and a non-subscriber.
2. Create ontology terms in the subscriber program.
3. Archive some of the ontology.
4. Have the sharer share with the subscriber and non-subscriber.
5. Have the subscriber subscribe and the non-subscriber not subscribe.
6. Verify that the ontology page for subscriber shows the same active and archived ontology.
7. Verify that the ontology page for non-subscriber does not show the same.
8. Edit a trait in the sharer program and verify that the change appears in the subscriber program.
9. Verify that the edit option is absent from trait details when viewed in the subscriber program.
10. Verify that the ontology import tab appears for sharer and non-subscriber but not for subscriber.
11. Verify that subscriber's ontology page displays under the title that it is sharing ontology from the sharer program.

# Checklist:
